packages/react-data/src/fetch/ReactDataFetch.js

```diff
@@ -120,16 +125,24 @@ Response.prototype = {
 };
 
 function preloadResult(url: string, options: mixed): Result {
-  const map = readResultMap();
+  const cache = readCache();
+  const map = readResultMap(cache);
   let entry = map.get(url);
   if (!entry) {
     if (options) {
       if (options.method || options.body || options.signal) {
-        // TODO: wire up our own cancellation mechanism.
         // TODO: figure out what to do with POST.
         throw Error('Unsupported option');
       }
     }
+    if (hasAbortController) {
+      const controller = new AbortController();
```

Comment:
Maybe I could create one controller per Cache Map entry? Seems unnecessary to create many of them if they're all going down together.